opensrp-reveal/build.gradle

```diff
@@ -177,15 +177,15 @@ tasks.withType(Test) {
 
 dependencies {
 
-    implementation('io.ona.kujaku:library:0.6.1') {
+    implementation('io.ona.kujaku:library:0.6.2') {
         transitive = true
         exclude group: 'com.android.volley'
         exclude group: 'kujaku', module: 'utils'
     }
 
     implementation 'io.ona.kujaku:utils:0.6.1'
```

Comment:
Increment the updated utils to 0.6.2

Reply:
Utils are not being released to a higher version.